src/games/strategy/net/MacFinder.java

```diff
@@ -14,20 +14,6 @@
 import games.strategy.util.MD5Crypt;
 
 public class MacFinder {
-  // For quick testing
```

Comment:
The test data here is of moderate quality, but at the same time it is quite trivial to get examples Mac Addresses. The outputs on these statements are not defined, so in the end we only have examples of trivial to get test data